airbyte-webapp/src/views/Connection/ConnectionForm/components/ScheduleField.tsx

```diff
@@ -0,0 +1,187 @@
+import { Field, FieldInputProps, FieldProps, FormikProps } from "formik";
+import { ChangeEvent, useMemo } from "react";
+import { useIntl } from "react-intl";
+
+import { ControlLabels, DropDown, DropDownRow, Input, Link } from "components";
+
+import { ConnectionScheduleData, ConnectionScheduleType } from "core/request/AirbyteClient";
+
+import availableCronTimeZones from "../../../../config/availableCronTimeZones.json";
+import { ConnectionFormMode } from "../ConnectionForm";
+import { FormikConnectionFormValues, useFrequencyDropdownData } from "../formConfig";
+import styles from "./ScheduleField.module.scss";
+
+interface ScheduleFieldProps {
+  scheduleData: ConnectionScheduleData | undefined;
+  mode: ConnectionFormMode;
+  onDropDownSelect?: (item: DropDownRow.IDataItem) => void;
+}
+
+const CRON_DEFAULT_VALUE = {
+  cronTimeZone: "UTC",
+  // Fire at 12:00 PM (noon) every day
+  cronExpression: "0 0 12 * * ?",
+};
+
+const CRON_REFERENCE_LINK = "https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm";
```

Comment:
I found out that we are using the `quartz` library on the backend in order to process cron expressions, and they have a tutorial page on cron expressions that I think is a bit more helpful than the oracle page. Could we link to that one instead? http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html

The other nice thing about linking there is that we can be more confident that their documentation matches our implementation, since we are using their library to process cron expressions.

Reply:
Yeah sure makes sense. Will change it.